qiskit_algorithms/eigensolvers/vqd.py

```diff
@@ -94,19 +94,14 @@ class VQD(VariationalAlgorithm, Eigensolver):
                 overlap estimation as indicated in the VQD paper.
             ansatz (QuantumCircuit): A parameterized circuit used as ansatz for the wave function.
             optimizer(Optimizer | Sequence[Optimizer]): A classical optimizer or a list of optimizers,
-            one for every k-th eigenvalue. Can either be a Qiskit optimizer or a callable
+            	one for every k-th eigenvalue. Can either be a Qiskit optimizer or a callable
                 that takes an array as input and returns a Qiskit or SciPy optimization result.
             k (int): the number of eigenvalues to return. Returns the lowest k eigenvalues.
             betas (list[float]): Beta parameters in the VQD paper.
                 Should have length k - 1, with k the number of excited states.
                 These hyper-parameters balance the contribution of each overlap term to the cost
                 function and have a default value computed as the mean square sum of the
                 coefficients of the observable.
-            initial point (Sequence[float] | Sequence[Sequence[float]] | None): An optional initial
```

Comment:
Why is this removed? The Attributes listed here are the public instance vars that can be directly read/written by an user. The initial point in the constructor is just a parameter, with the same name, that ends up being the initial value for this. So it needs the documentation in both places - I see it was done like this in qiskit.algorithms but to me its wrong for this to be deleted - though it should be the correct name i.e `initial_point` as it seems to be missing the underscore. Can we put out back and with the underscore and go from there - its not clear to me why this would be special given there are other attributes here that are also same as init parameters

Reply:
I agree that the best solution would be to only correct the name of the attribute. The difference between this attribute and the rest is that we have it duplicated as a property in lines 170-178. Because of that, Sphinx complains about it if we only try to fix the name, as @ElePT pointed out in this [comment](https://github.com/Qiskit/qiskit/pull/12162#discussion_r1557767059). Given that removing the docs won't remove the attribute entirely from the class page, we thought that it was a good solution. Do you think is it enough for the attribute to be [here](https://qiskit-community.github.io/qiskit-algorithms/stubs/qiskit_algorithms.VQD.html#qiskit_algorithms.VQD.initial_point)?

Reply:
Ah ok, I see, there is even a comment there that this is a setter/getter to conform to the Variational interface. So initial_point is exposed via that setter and getter, and unlike the others is not a public instance var. Guess I did not look carefully enough to notice that, in which case then things are fine. Sorry about that. I see it shows up under Attributes separate from that other list as a consequence of it being a getter/setter.Thanks for fixing this here too.

As @ElePT did the other I'll let her look here too but it seems fine to me after your explanation.

Reply:
FYI I labelled it for backport as we can upload/refresh the docs, when its just a docs fix, outside of doing a code release.